### PR TITLE
Posenet Extension implementation

### PR DESCRIPTION
**Pull request by ellelili2025**
This is an enhancement, which allows video view of a person on Canvas while analyzing the pose , of original postnet extenstion implementation.

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java [hunk 1]
```diff
@@ -993,6 +1011,19 @@ public void BackgroundImage(String path) {
...
+  /**
+   * Specifies the back ground image in Base64 format
+   * imageUrl will be in format of:  data:image/png;base64,iVBORw0KG...s//f+4z/6Z
+   * @param imageUrl the base64 format for an image
+   */
+  @android.support.annotation.RequiresApi(api = android.os.Build.VERSION_CODES.FROYO)
+  @SimpleProperty (
+          description = "Set the back ground image in Base64 format"
+  )
+  public void BackGroundImageinBase64(String imageUrl) {
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java]: Keep the case consistent with the existing properties `BackgroundColor` and `BackgroundImage`.

Comment (ewpatton) [reply appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java]: Add a check that if the version code of the current build is less than FROYO that nothing happens, and also note this in the documentation.

**Comment by ewpatton** [comment]
@ellelili2025 Also, if I understand correctly, this requires that one feed the image from the Posenet extension back to the Canvas before drawing on it. I don't see where the drawing operations are separated from the background. Do we have to redraw everything on every frame?

**Comment by ellelili2025** [comment]
@ewpatton  Yes. The current implementation is that we update the image when we update pose so that the pose correlates with the image. So if one wants to update the pose on canvas, it is better to update the background image as well.

File: appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java (hunk 1 above)
Comment (ellelili2025) [reply appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java]: Addressed both comments.

**Comment by ewpatton** [comment]
The posenet extension should be in the extensions repo, and since the Canvas changes went into #2041 I think this PR is now no longer needed.

**Pull request closed without merging**